package.json

```diff
@@ -62,6 +62,7 @@
     "gatsby-transformer-sharp": "^2.5.3",
     "hex-rgb": "^4.1.0",
     "install": "^0.13.0",
+    "json5": "^2.1.3",
```

Comment:
Added json5 to support json verification for strings that didn't add quotation marks around some of the attributes.

The standard JSON.parse() was failing on any of the tests that were formatted as follows

` {
    user: {
       name: 'Samuel'
     }
}`